src/engines/mod.rs

```diff
@@ -578,10 +574,22 @@ fn input_seek<'a, I: 'a + IoProvider>(es: *mut ExecutionState<'a, I>, handle: *m
         libc::SEEK_SET => SeekFrom::Start(offset as u64),
         libc::SEEK_CUR => SeekFrom::Current(offset as i64),
         libc::SEEK_END => SeekFrom::End(offset as i64),
-        _ => panic!("Unexpected \"whence\" parameter to fseek() wrapper: {}", whence),
+        _ => {
+                // TODO: Handle the error better. This indicates a bug in the engine.
+                tt_warning!(es.status, "Unexpected \"whence\" parameter to fseek() wrapper: {}",
```

Comment:
Very minor, but while we're here, I'm trying to be more consistent about these sorts of messages not starting with capital letters.